build_platform.py

```diff
@@ -218,6 +211,13 @@ def run_or_die(cmd, error):
 
 print("Libraries installed: ", glob.glob(os.environ['HOME']+'/Arduino/libraries/*'))
 
+# link our library folder to the arduino libraries folder
+if not IS_LEARNING_SYS:
```

Comment:
symlink should be done after lib deps is installed and our_name library is uninstalled